File: server/tests/fixtures/alpha_rule_triggers/csharp/imports_cycle_advanced_b.cs
// Part of imports.cycle.advanced fixture - this file uses ImportsCycleAdvanced back
// Should trigger: imports.cycle.advanced
using System;

namespace Fixtures.CSharp
{
    public class ImportsCycleAdvancedB
    {
        public string FunctionB()
        {
            var a = new ImportsCycleAdvanced();
            return a.FunctionA() + " from B";
        }
    }
}

File: server/tests/fixtures/alpha_rule_triggers/java/imports_cycle_advanced_b.java
// Part of imports.cycle.advanced fixture - this file imports ImportsCycleAdvanced back
// Should trigger: imports.cycle.advanced
package fixtures.java;

import fixtures.java.ImportsCycleAdvanced;

public class ImportsCycleAdvancedB {
    public String functionB() {
        ImportsCycleAdvanced a = new ImportsCycleAdvanced();
        return a.functionA() + " from B";
    }
}
